regenerate_bindings.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
-bindgen --size_t-is-usize --no-recursive-whitelist --no-prepend-enum-name --no-layout-tests \
---generate "functions,types" --whitelist-function="snd_.*" --whitelist-type="_?snd_.*" \
-/usr/include/alsa/asoundlib.h -o src/generated.rs
+
+export FORCE_REGENERATE_BINDINGS=$RANDOM
+export BINDINGS_OUT_DIR=src/
+
```

Comment:
Naming: Maybe we should have our own namespace here, like `ALSA_SYS_REGENERATE_TOKEN` and `ALSA_SYS_BINDINGS_OUTPUT` where the latter is the entire filename, not just the path.